Angular/src/app/admin-table/admin-table.component.ts

```diff
@@ -17,109 +17,80 @@ export class AdminTableComponent implements OnInit {
   capabilities = [];
   jobsInDep: any;
   bands: any;
-  id: any;
+  departmentId: any;
   alertMessage: string;
   alertType: string;
   showAlert = false;
 
   constructor(
     private dataService: DataService,
-    private modalService: NgbModal,
-    private router: Router
+    private modalService: ModalService,
   ) { }
 
   ngOnInit(): void {
     var urlParams = new URLSearchParams(window.location.search);
-    this.id = parseInt(urlParams.get("id"));
+    this.departmentId = parseInt(urlParams.get("id"));
     this.loadRoles();
   }
 
+  async switchModal(selectedRole) {
+    if(selectedRole.ID == -1){
```

Comment:
Generally good to use `===`. It checks if the values are equal, and also if the types are equal